### PR TITLE
Mark outbound rule properties as required

### PR DESCRIPTION
**Pull request by khannarhea**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by christiankuhtz** [comment]
@khannarhea looks good to me.

**Comment by MikhailTryakhov** [comment]
@christiankuhtz @khannarhea this is a breaking change. by some reason SDK is still not releseased and we'll include this info there. but next time it'll be very bad to push such changes in next SDK version it'll cause problems. Can we be sure your swagger is ready?

**Review by MikhailTryakhov — APPROVED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json [hunk 1]
```diff
@@ -1459,7 +1459,9 @@
         }
       },
       "required": [
-        "backendAddressPool"
```
Comment (sergey-shandar) [inline specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json]: @khannarhea @MikhailTryakhov why the changes for `2018-07-01` are targeting September branch (2018-08-01)? IMHO, it should target `master`.

**Review by sergey-shandar — CHANGES_REQUESTED** [review]

File: specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json (hunk 1 above)
Comment (MikhailTryakhov) [reply specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json]: @sergey-shandar  you're right. @khannarhea  please retarget it to ```master``` branch

**Comment by sergey-shandar** [comment]
I'm closing this PR in favor of https://github.com/Azure/azure-rest-api-specs/pull/3697
